docker-compose.yml

```diff
@@ -6,28 +6,33 @@ services:
   web:
     image: webdevops/php-apache-dev:${PHP_VERSION}
     container_name: web
-    user: www-data
+    user: "root"
```

Comment:
We are replacing rsync with volumes of docker.
We want the project files to still be owned by the root user of the container (1000:1000) of the host while still giving permissions to www-data to run and write on the project.
To avoid giving these permissions we should run the apache web as root, which is the host user.
With these changes the plugin is available to the magento container through the docker volumes.

Reply:
@Aitor-Corrales What do you think about this change?